packages/slate-editor/src/components/Menu/Dropdown.module.scss

```diff
@@ -132,54 +132,4 @@
             }
         }
     }
-
-    .editor-menu-dropdown__menu-option {
-        margin: 0;
-        font-size: 13px;
-        color: inherit;
-
-        &--paragraph {
-            // Override margin set on `.editor-story p`.
-            margin-bottom: 0 !important;
-        }
-
-        &--heading-1 {
-            font-size: 17px;
-            font-weight: 700;
-        }
-
-        &--heading-2 {
-            font-size: 15px;
-            font-weight: 700;
-        }
-
-        &--unordered-list,
-        &--ordered-list {
-            padding-left: $spacing-2;
-        }
-
-        &--unordered-list {
-            // Fix broken list style. Breaks because this list
-            // is nested in another list (menu items).
-            list-style-type: disc;
-        }
-
-        &--blockquote {
-            padding: 0;
-            background: unset;
-        }
-
-        &--blockquote > p {
-            padding-left: $spacing-half;
-            font-style: italic;
-        }
-
-        &--blockquote > p::before {
```

Comment:
This part was a spaghetti blob mixed with the code located on a different abstraction level. Fixed by moving the styles around to where they belong.